rules/game_process.tex

```diff
@@ -90,10 +89,9 @@ \subsection{Robot States}
 Once the game controller operator has confirmed the \texttt{standby} state, the referee should wait a random time between 10 to 120 seconds before announcing the transition to \texttt{ready}. 
 Teams should expect that the delay before the referee signals the transition from \texttt{standby} to \texttt{ready} will be different every time.
 To announce the transition from \texttt{standby} to \texttt{ready} state, the referee will raise both hands over their head.
```

Comment:
Since we are introducing multiple different visual signals, perhaps we could utilise some diagrams for the visual signal. Similar to what was used for the 2023 challenge